spec/rails_helper.rb

```diff
@@ -31,8 +31,10 @@ module Features
 end
 
 ActiveRecord::Migration.maintain_test_schema!
+
 Capybara.javascript_driver = :poltergeist
 Capybara.register_driver :poltergeist do |app|
   options = { phantomjs_options: ["--load-images=no"] }
   Capybara::Poltergeist::Driver.new(app, options)
 end
+Capybara.server = :webrick
```

Comment:
Currently, CI fails because it can't find Puma. Should we instead be adding that, rather than switching to Webrick here? If I recall correctly, Webrick won't execute any Javascript, which we may have in some tests?

Reply:
As I understand, `Capybara.server` is just the server, not js-driver.
Maybe there is some performance difference, so I agree this should be replaced with Puma.